src/glabs-inline/web/index.scss

```diff
@@ -68,7 +73,7 @@ h1, p {
 .body__tagline {
   font-size: 10px;
   color: $neutral-2;
-  margin-top: 4px;
+  margin-top: auto;
```

Comment:
Within a flex container, it sets as large a margin as possible, i.e. it sits flush against the bottom of the div.